var/spack/repos/jcsda-emc/packages/GFDL_atmos_cubed_sphere/CMakeLists.txt.patch

```diff
@@ -0,0 +1,11 @@
+--- a/CMakeLists.txt
```

Comment:
No longer required for Skylab v8 because of https://github.com/JCSDA/GFDL_atmos_cubed_sphere/pull/3